okio/src/main/kotlin/okio/Base64.kt

```diff
@@ -143,5 +143,5 @@ internal fun ByteArray.encodeBase64(map: ByteArray = BASE64): String {
       out[index] = '='.toByte()
     }
   }
-  return String(out, Charsets.US_ASCII)
-}
\ No newline at end of file
+  return out.toAsciiString()
```

Comment:
You can use UTF-8 here. The behavior is the same!

Reply:
Yep, true